### PR TITLE
Add option to output cache age header

### PR DESCRIPTION
**Pull request by it-can**
I find it handy to have an cache age header in my responses. This option is disabled by default, and only works when the time-header is also active

### CODE REVIEW

File: src/Middlewares/CacheResponse.php [hunk 1]
```diff
@@ -31,6 +32,12 @@ public function handle(Request $request, Closure $next, ...$args): Response
 
                 $response = $this->responseCache->getCachedResponseFor($request, $tags);
 
+                // Add cache age header
```
Comment (freekmurze) [inline src/Middlewares/CacheResponse.php:35]: Could move this functionality in a function with a good name, so we can lose that comment?

**Review by freekmurze — CHANGES_REQUESTED** [review]
Could you also update the readme?

File: src/Middlewares/CacheResponse.php (hunk 1 above)
Comment (it-can) [reply src/Middlewares/CacheResponse.php]: done!

**Comment by it-can** [comment]
> Could you also update the readme?

done

**Pull request merged**